app/controllers/teachers/schools_controller.rb

```diff
@@ -7,9 +7,15 @@ class SchoolsController < ApplicationController
 
     def show
       redirect_to enrol_path unless @school.active? || (current_user && current_user.manages_school?(@school.id))
-      @charts = [:teachers_landing_page_electricity, :teachers_landing_page_gas]
-      setup_find_out_more
-      setup_activity_suggestions
+
+      if AggregateSchoolService.new(@school).in_cache? || ! Rails.application.config.action_controller.perform_caching
```

Comment:
I don't like asking whether caching is switched on, but of courses tests, don't have it switched on...